wordpress-mobile/WordPress-iOS
# Blogging Prompts Feature Introduction: add header image

Ref: #18176 

An optional header image has been added to `CollapsableHeaderViewController`.
- The lightbulb image for Blogging Prompts' feature introduction is now displayed. 
  - Currently it is simply orange. Gradient colors will be added in future.
- The header image is not shown for other implementations of `CollapsableHeaderViewController`.

NOTE: The Feature Introduction header is still not shown in compact height. Coming soon!

To test:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction will appear. 
- Verify the lightbulb icon appears above the header title.
- Go to either of these and verify the header image is not shown.
  - FAB > Site page
  - Create WP site

| ![fi](https://user-images.githubusercontent.com/1816888/162083637-c81e5810-1b61-43e9-b837-97514814888a.png) | ![site_page](https://user-images.githubusercontent.com/1816888/162083626-3d9bf050-5a45-429a-9a8f-1a800f181acc.png) |
|--------|-------|

## Regression Notes
1. Potential unintended areas of impact
The image is only shown if provided, so there should be no impact on existing `CollapsableHeaderViewController` implementations.

3. What I did to test those areas of impact (or what existing automated tests I relied on)
I checked. All good.

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
